frontend/base/src/main/java/com/virtuslab/gitmachete/frontend/resourcebundles/GitMacheteBundle.java

```diff
@@ -26,12 +29,17 @@ private GitMacheteBundle() {}
    *             but each parameter must be a non-null {@link String} and not just a nullable {@link Object}
    * @return the formatted string
    */
-  public static String format(@I18nFormatFor("#2") String format, String @MinLen(1)... args) {
+  public static @PolyTainted String format(@PolyTainted @I18nFormatFor("#2") String format, String @MinLen(1)... args) {
     return MessageFormat.format(format, (@Nullable Object[]) args);
   }
 
   @I18nMakeFormat
-  public static String getString(@PropertyKey(resourceBundle = BUNDLE) String key) {
+  public static @Tainted String getString(@PropertyKey(resourceBundle = BUNDLE) String key) {
```

Comment:
`@Tainted` is kinda the implicit default for everything in the entire codebase, here only added for clarity